### ISSUE TITLE
Lower the full screen resolution

### ISSUE THREAD

**Issue by SebasMCYT** [TurboWarp/packager#654]
My game lags a bit on high resolutions so I thought: Is there a way to lower the full screen resolution? I know that in windowed mode you can just make the window smaller and that will lower the resolution, but it would be nice to run the game in full screen with lower resolution, it could help make the game run better on low-end computers. Is there any custom js/css to do this?

**Comment by TheShovel** [comment]
Disable high quality pen.

**Comment by SebasMCYT** [comment]
> Disable high quality pen.

Already disabled

**Comment by TheShovel** [comment]
Then there is nothing you can do from the project to fix the issue. You can either:
A. Lower you display resolution
B. Play in windowed mode

Also, could you send me a picture of your task manager, with the info for your hardware? (Or neofetch if you are on linux)

**Comment by SebasMCYT** [comment]
![image](https://user-images.githubusercontent.com/126848249/222791848-74c59919-9879-4fa8-a95a-abcecaa5a3cd.png)
Intel(R) Core(TM) i3-3240 CPU @ 3.40GHz
Intel HD Graphics
12.0 GB (11.9 GB) RAM
64-bit operating system, x64-based processor
1366x768

**Comment by TheShovel** [comment]
Your cpu is very underpowered. I am sorry but there is nothing you can do.

**Comment by SebasMCYT** [comment]
Yeah I thought so, thanks anyway, hope it works out better for others.
